config/software/datadog-agent-integrations.rb

```diff
@@ -4,6 +4,7 @@
 
 dependency 'pip'
 dependency 'datadog-agent'
+dependency 'unixodbc'
```

Comment:
is this meant to be added to the Windows and macOS builds as well? (https://github.com/DataDog/datadog-agent/pull/2729 only applied to Linux)